### ISSUE TITLE
:not(selector) expand error?

### ISSUE THREAD

**Issue by bokuweb** [rtsao/csjs#42]
Hi.I wrote the following style with csjs v.1.0.3

``` js
module.exports = csjs`
@media screen and (min-width: 769px) {
  .columns:not(.is-desktop) {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}
`
```
But it is expanded as follows

``` js
module.exports = csjs`
@media screen and (min-width: 769px) {
  .columns_2B2Wm:not(.is-desktop)_2B2Wm {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}
`
```

**Comment by rtsao** [comment]
So aside from the incorrect appending of hash outside the parens, is the `.is_desktop` class meant to be a global (unscoped)?

**Comment by bokuweb** [comment]
`.is-desktop` is scoped.

``` js
.columns_1oLjZ5.is-desktop_1oLjZ5 {
  -webkit-box-flex: 0;
  -webkit-flex: none;
      -ms-flex: none;
          flex: none;
}

@media screen and (min-width: 769x) {
  .columns_1oLjZ5.is-desktop_1oLjZ5 {
    display: block;
  }
}

@media screen and (min-width: 980px) {
  .columns_1oLjZ5:not(.is-desktop)_1oLjZ5 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}
```

**Comment by rtsao** [comment]
Fixed in `v1.0.4`. Thanks for the bug report!

**Comment by bokuweb** [comment]
wow!! thanks!!!
